### ISSUE TITLE
Move twisted.python._dist3 into _setup

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8852]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8852|
|Type|enhancement|
|Created|2016-10-05 23:50:01Z|

This is an implementation detail, and for [#8817](https://github.com/twisted/twisted/issues/8817) it should be combined.

<details><summary>Searchable metadata</summary>

```
trac-id__8852 8852
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_5 Twisted 16.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1475711401893451 1475711401893451
changetime__1476058428184453 1476058428184453
version__None None
owner__Mark_Williams__markrwilliams_____ Mark Williams <markrwilliams@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

Tests are at a reasonable state of passing. Please review.

PR: https://github.com/twisted/twisted/pull/555

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_2"></a>|@markrwilliams commented|
|-|-|

The diff is big but the change is simple and [it needs to happen](https://github.com/twisted/twisted/pull/545#issuecomment-249102734).  I'm going to merge this.  Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_3"></a>|@markrwilliams set owner to @markrwilliams<br>@markrwilliams set status to `closed`|
|-|-|

In changeset ec42d7209d98f36b89b114520558d34a4034a44f
```
#!CommitTicketReference repository="" revision="ec42d7209d98f36b89b114520558d34a4034a44f"
Merge pull request #555 from twisted/8852-dist3-rm

Author: hawkowl
Reviewer: markrwilliams
Fixes: #8852

Move _dist3 into _setup
```
